src/client/src/translations/whitelist_en.json

```diff
@@ -16,7 +16,6 @@
   "summaryTile.changeItemName",
   "topNavBar.ariaNavLabel",
   "topNavBar.frameworks",
-  "topNavBar.pages",
```

Comment:
This diff comes from changes done in a different PR right? Maybe I'm misunderstanding something, but if so then let's try to keep the translations changes on the PR that actually changes them.